src/torchcodec/decoders/_core/VideoDecoder.cpp

```diff
@@ -817,6 +869,30 @@ void VideoDecoder::setCursorPtsInSeconds(double seconds) {
   desiredPtsSeconds_ = seconds;
 }
 
+bool VideoDecoder::canWeAvoidSeekingAudio(double desiredPtsSeconds) const {
+  const StreamInfo& streamInfo = streamInfos_.at(activeStreamIndex_);
+  int64_t targetPts = *desiredPtsSeconds_ * streamInfo.timeBase.den;
```

Comment:
Nit: we should probably use `secondsToClosestPts()` unless there's an explicit reason we don't want to use `std::round()`.